docs/fundraising/paypal-button.md

```diff
@@ -1,8 +1,17 @@
-## Custom PayPal Buttons
-Chapters have the option of embedding a PayPal donation button on their website. (This option isn't available for Independent Communities.)
+# Custom PayPal Buttons
+Chapters, Bridges, and Programs have the option of embedding a PayPal donation button on their website. (This option isn't available for Independent Communities.)
 
-Request a PayPal button by completing [this form](https://docs.google.com/forms/d/e/1FAIpQLSd3q_rYZ1U8464QBMeWYvvno06VLob5eS6tqjm2HCSLXBbStQ/viewform?usp=sf_link). The form can be submitted by any registered Volunteer Fundraiser for your Chapter. 
+Request a PayPal button by completing [this form](https://docs.google.com/forms/d/e/1FAIpQLSd3q_rYZ1U8464QBMeWYvvno06VLob5eS6tqjm2HCSLXBbStQ/viewform?usp=sf_link). The form can be submitted by any registered Volunteer Fundraiser for your Chapter, Bridge or Program. 
 
 Prior to requesting a button, decide whether you want it to redirect back to your website after the donor pays or not. A best practice is to create a donation thank you page on your website, and have the button redirect there after the donation. If you go this route, you will need to set up this page on your website prior to requesting a button.
 
 You will receive your new PayPal button code via email within a week (usually less). If you don't see the button come through within a few days, feel free to email us at finance@bridgefoundry.org.
+
+## Unrestricted Funds language
+When you embed the code on your website, we require that you include some language about unrestricted funds immediately above, below, or next to the button so that donors will see it. This is important so that donors understand how their donation will be used and we avoid any misunderstandings. There are two options (short and long) for this language - pick whichever one you prefer, but either way please do include the link to our unrestricted funds policy as listed below.
+
+Short version:
+"All donations are provided to Bridge Foundry, Inc. as [unrestricted funds](https://rebrand.ly/bridge-foundry-unrestricted-funds)."
```

Comment:
Why are the links through rebrand.ly instead of just to https://operations.bridgefoundry.org/fundraising/unrestricted-funds/
?

Reply:
In case the link changes. If, for example, we tweak the hierarchy in the ops guide and it becomes a different URL, I can update the rebrandly link in just one spot and then we don't have broken links in a bunch of places that we need to fix. It has already proved useful as we've moved some things around.

Reply:
Neat, thanks.